src/main/java/com/fortanix/sdkms/performance/helper/AESHelper.java

```diff
@@ -40,10 +40,17 @@
     @Override
     public EncryptRequestEx createEncryptRequest(SobjectDescriptor key, String plain) {
         ByteArrayOutputStream input = new ByteArrayOutputStream();
+        if(mode == CryptMode.FPE) {
+            plain = "36088650107272";
```

Comment:
Why are we overriding the plain value? This means user won't be able to set a custom value.

Reply:
Taken care of.